public/r2/v1/swagger/swagger.json

```diff
@@ -184,6 +184,79 @@
                 }
             }
         },
+        "/namespaces/{namespaceID}/ruleset/bulk": {
+            "put": {
+                "tags":[
+                    "namespaces"
+                ],
+                "summary": "Bulk update endpoint for namespace changes",
+                "operationId": "bulkUpdateRuleSet",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "in": "path",
+                        "name": "namespaceID",
```

Comment:
This is probably unnecessary to be required. You are updating the resource which is in the URL, so this can be inferred.

Reply:
Interesting. I took a look at the other endpoint URIs and it looked like they required parameters that were part of the URL. Perhaps we should keep if for consistency.

Reply:
If thats the case, then lets keep it consistent